src/graphql/queries/GetSuggestedWorkForLabware.graphql

```diff
@@ -0,0 +1,10 @@
+query GetSuggestedWorkForLabware($barcodes: [String!]!) {
+  suggestedWorkForLabware(barcodes: $barcodes) {
+    suggestedWorks {
+      ...SuggestedWorkFields
+    }
+    works {
```

Comment:
I don't think you need to get the `works` stuff here at all: as far as I can tell you only use the `suggestedWorks`.

Reply:
Updated. Thank you